internalsdk/pro/pro.go

```diff
@@ -60,48 +91,46 @@ func NewClient(baseURL string, opts *Opts) ProClient {
 	client := &proClient{
 		userConfig: opts.UserConfig,
 	}
-	client.webclient = webclient.NewRESTClient(defaultwebclient.SendToURL(httpClient, baseURL, client.setUserHeaders(), nil))
```

Comment:
I think it would be cleaner to continue setting the default headers a single time in request middleware vs. having to pass the same headers every time we call a method on RESTClient.

If the headers differ between ProClient and AuthClient, what about creating two separate clients instead? I will open a PR with some suggestions..

Reply:
https://github.com/getlantern/lantern-client/pull/1101

Reply:
thanks, Paul, I will take a look at it shortly.

Reply:
So the only reason I expose the header is, In the future, we need to change headers for specific endpoints we can do this easily, without a lot of changes, Before the header was passing it tightly coupled we didn't have any flexibility on that.